### PR TITLE
RoD command update

### PR DESCRIPTION
**Pull request by Haxxz**
# PvME Change Submission
Thank you for helping maintain our resources! Below is a quick sanity checklist that helps make sure we can keep track of what is changing. Please fill it out :)

## Sanity Checks
- [i hope so ] - the title of this pull request clearly describes the change I would like to make.
- [single change ] - if there are multiple changes in this pull request, they are all related.
- [ i hope so (RoD command update)] - I have tried to write clear a commit message(s) that describes the changes I made.

### CODE REVIEW

File: commands/rod.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
       {
         "name": "__General Information & Benefits__",
-        "value": "*Note: more details available [here](https://runescape.wiki/w/Ring_of_Death)*\n⬥ **Effect**: revived to full HP, 3 min cooldown; activates before Sign of Life <:signoflife:1005552211847295076>\n\u00a0\u00a0\u00a0\u00a0• A ramping DoT is applied for 3 minutes.\n\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0- Taking off <:RoD:513190159462825984> triples the damage until it is re-equipped\n⬥ Effect is removed on many phase transitions → can be used near-losslessl.\n⬥ Can be overcharged to have 5x capacity by Death.\n⬥ Upgradeable at the Mazcab Armoursmith to increase its damage bonus"
+        "value": "*Note: more details available [here](https://runescape.wiki/w/Ring_of_Death)*\n⬥ **Effect**: revived to full HP, 3 min cooldown; activates after Sign of Life <:signoflife:1005552211847295076>\n\u00a0\u00a0\u00a0\u00a0• A ramping DoT is applied which lasts until death or return to a bank area.\n\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0- Taking off <:RoD:513190159462825984> triples the damage until it is re-equipped.\n⬥ Upgradeable at the Mazcab Armoursmith to increase its damage bonus"
```
Comment (nullopt) [inline commands/rod.txt]: I would probably keep the 5x capacity section, but mention that it is not recommended as it is no longer worth it.

**Review by nullopt — CHANGES_REQUESTED** [review]

File: commands/rod.txt (hunk 1 above)
Comment (Haxxz) [reply commands/rod.txt]: Okay

Comment (Haxxz) [reply commands/rod.txt]: Removed suggested secondary (adren gaining) effect and condensed as suggested by Archeus

**Review by nullopt — APPROVED** [review]
LGTM

**Pull request merged**